docs/operations/migration_policies.md

```diff
@@ -0,0 +1,182 @@
+yet and that APIs may change in the future.
+
+## Overview
+
+KubeVirt supports [Live Migrations](./live_migration.md) of Virtual Machine workloads.
+Before migration policies were introduced, migration settings could be configurable only on the cluster-wide
+scope by editing [KubevirtCR's spec](https://kubevirt.io/api-reference/master/definitions.html#_v1_kubevirtspec)
+or more specifically [MigrationConfiguration](https://kubevirt.io/api-reference/master/definitions.html#_v1_migrationconfiguration)
+CRD.
+
+Several aspects (although not all) of migration behaviour that can be customized are:
+- Bandwidth
+- Auto-convergence
+- Post/Pre-copy
+- Max number of parallel migrations
+- Timeout
+
+Migration policies generalize the concept of defining migration configurations, so it would be
+possible to apply different configurations to specific groups of VMs.
+
+Such capability can be useful for a lot of different use cases on which there is a need to differentiate
+between different workloads. Differentiation of different configurations could be needed because different
+workloads are considered to be in different priorities, security segregation, workloads with different
+requirements, help to converge workloads which aren't migration-friendly, and many other reasons.
+
+## API Examples
+
+#### Migration Configurations
+Currently, the MigrationPolicy spec will only include the currently relevant configurations from KubevirtCR's
+MicrationConfiguration:
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+    allowAutoConverge: true
+    bandwidthPerMigration: 217Ki
+    completionTimeoutPerGiB: 23
+    allowPostCopy: false
+```
+
+All above fields are optional. When omitted, the behaviour that will apply is the currently defined behaviour in
+KubevirtCR's MigrationConfiguration. This way, KubevirtCR will serve as a configurable set of defaults for both
+VMs that are not bound to any MigrationPolicy and VMs that are bound to a MigrationPolicy that does not
+define all the configurations.
+
+#### Matching Policies to VMs
+
+Next in the spec are the selectors that define the group of VM on which to apply the policy. The options to do so
+are the following.
+
+**This policy applies to the VMs in namespaces that have all the required labels:**
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+  selectors:
+    namespaceSelector:
+      matchLabels:
+        hpc-workloads: true       # Matches a key and a value 
+        xyz-workloads-type: ""    # Matches the key only, any value applies
+```
+
+**This policy applies for the VMs that have all the required labels:**
+```yaml
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: MigrationPolicy
+  spec:
+  selectors:
+    virtualMachineInstanceSelector:
+      matchLabels:
+        workload-type: db       # Matches a key and a value 
+        operating-system: ""    # Matches the key only, any value applies
```

Comment:
nit: very late to the game, I know, but from a readability POV I stumbled over the empty string here. At first I figured it might mean "the label is NOT present". Maybe something like "*" would have been better here?

BTW is something like `"!label_name"` possible here?

Reply:
This is a great catch actually.
As you're pointing out, this is confusing. For this and other reason this feature isn't merged yet and isn't planned to get merged in the near future.

I'll remove this from the user-guide, thanks for catching that!